lib/actions.js

```diff
@@ -153,6 +153,22 @@ exports.type = function() {
 };
 
 /**
+ * Populate a text into an element
+ *
+ * @param {String} selector
+ * @param {String} text
+ * @param {Function} done
+ */
+
+exports.populate = function(selector, text, done) {
+  debug('populate() %s into %s', text, selector);
+  this.evaluate_now(function(selector, text) {
+    var element = document.querySelector(selector);
+    element.value = text;
```

Comment:
In 2.0.9 (I think) the old version of type() also called element.focus() before and element.blur() after setting the value here.